packages/beacon-node/src/network/reqresp/beaconBlocksMaybeBlobsByRoot.ts

```diff
@@ -637,6 +639,7 @@ export async function unavailableBeaconBlobsByRootPostFulu(
       dataColumnsSource: DataColumnsSource.byRoot,
     } as BlockInputDataColumns;
     resolveAvailability(blockData);
+    opts.metrics?.dataColumns.bySource.inc({source: blockData.dataColumnsSource}, blockData.dataColumns.length);
```

Comment:
This collection point should be moved up just below the call to `sendDataColumnSidecarsByRoot` with a source of `ByRoot`.